proto/commonfate/access/v1alpha1/audit_logs.proto

```diff
@@ -64,4 +86,14 @@ message AuditLog {
   // such as the particular authentication token ID used
   // to perform the action.
   repeated IdentityLink caller_identity_chain = 8;
+
+  //the request id relating to the audit log
+  string request_id = 9;
+
+  AuditRoleEntitlement entitlement = 10;
```

Comment:
The entitlement should be optional, so that we can support audit logs which do not relate to grants or requests